models/experimental/functional_stable_diffusion3_5/ttnn/ttnn_patch_embed.py

```diff
@@ -0,0 +1,122 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import ttnn
+
+from models.experimental.functional_stable_diffusion3_5.ttnn.common import Conv
+
+
+class ttnn_PatchEmbed:
```

Comment:
Follow PEP8 for naming (TitleCase for classes)

Reply:
Okay, I will update it.